src/DashboardHub/Bundle/AppBundle/Entity/Dashboard.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace DashboardHub\Bundle\AppBundle\Entity;
+
+/**
+ * Class Dashboard
+ * @package DashboardHub\Bundle\AppBundle\Entity
+ */
+class Dashboard
+{
+
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var User
+     */
+    protected $user;
+
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @var string
+     */
+    protected $repo;
```

Comment:
repo? funny you let that past seeing as you are so pedantic :)

Is repository not more accurate? Also will this ever be a collection?